src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,47 @@
+
+.container {
+  display: flex;
+  margin-top: 16px;
+
+}
+
+.stars {
+  display: flex;
+
+  &__star {
+    margin-left: 4px;
+    width: 16px;
+    height: 16px;
+    border-radius: 0.5px;
+    background-image: url(../images/star.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+    list-style: none;
+  }
+
+  &__star:first-child {
+    margin-left: 0;
+  }
+
+  @for $starCount from 1 through 5 {
+    &--#{$starCount} .stars__star:nth-child(-n + #{$starCount}) {
+      background-image: url(../images/star-active.svg);
+    }
+  }
+
+  &-reviews {
```

Comment:
The class 'stars-reviews' should follow BEM naming conventions and be an element of the 'stars' block, e.g., 'stars__reviews'.